pages/implementation/avo-codegen-overview.mdx

```diff
@@ -37,7 +39,7 @@ When designing the tracking plan you configure the list of data destinations for
 
 In the Avo Tracking Plan you can configure which analytics <Link passHref href="/workspace/tracking-plan/events#a-nameactionsa-actions"><a>actions</a></Link> should be included in Codegen (for example you might want to update user properties or log revenue along with logging an event), and Codegen will handle calling all the relevant Destination Interface methods. All you have to do is call the generated function in your code base and pass in type safe key-value pairs, and the Codegen will take care of the rest.
 
-Learn more technical details about Codegen <Link passHref href="/implementation/avo-functions-overview"><a>here</a></Link>.
+Learn more technical details about Codegen <Link passHref href="/implementation/avo-codegen-tech-deep-dive"><a>here</a></Link>.
```

Comment:
I'm wondering if we can somehow make the hyperlink not reference "here" ? 🤔

"Read the <Link> technical details on Codegen </Link> or something, no strong opinions on this

Reply:
Good call. Done!